`mypy` complains about new `azure-storage-blob` package

...that was released yesterday (https://pypi.org/project/azure-storage-blob/12.9.0/).

Copied from CI log:
```
[...]/python3.8/site-packages/azure/storage/blob/_serialize.py:129: error: Type signature has too many arguments  [syntax]
[...]/python3.8/site-packages/azure/storage/blob/_blob_client.py:1407: error: Type signature has too few arguments  [syntax]
[...]/python3.8/site-packages/azure/storage/blob/aio/_blob_client_async.py:808: error: Type signature has too few arguments  [syntax]
webviz_config/_deployment/azure_cli.py:244: error: "ContainerClient" has no attribute "__aenter__"  [attr-defined]
webviz_config/_deployment/azure_cli.py:244: error: "ContainerClient" has no attribute "__aexit__"  [attr-defined]
```

--- a/mypy.ini
+++ b/mypy.ini
@@ -1,0 +1,8 @@
+[mypy]
+disallow_untyped_defs = True
+show_error_codes = True
+warn_unused_ignores = True
+
+# https://github.com/Azure/azure-sdk-for-python/issues/20771
+[mypy-azure.storage.blob.*]
+ignore_errors = True

--- a/webviz_config/__init__.py
+++ b/webviz_config/__init__.py
@@ -1,6 +1,6 @@
 try:
     # Python 3.8+
-    from importlib.metadata import version, PackageNotFoundError  # type: ignore
+    from importlib.metadata import version, PackageNotFoundError
 except ModuleNotFoundError:
     # Python < 3.8
     from importlib_metadata import version, PackageNotFoundError  # type: ignore

--- a/webviz_config/_config_parser.py
+++ b/webviz_config/_config_parser.py
@@ -170,7 +170,7 @@ def _call_signature(
                         mapped_args[arg] = value
                         break
 
-            result = deprecation.callback(**mapped_args)  # type: ignore
+            result = deprecation.callback(**mapped_args)
             if result:
                 argument_deprecation_warnings.append(result[0])
                 _values = [

--- a/webviz_config/_deployment/azure_cli.py
+++ b/webviz_config/_deployment/azure_cli.py
@@ -241,7 +241,7 @@ def storage_container_upload_folder(
                 )
 
     async def _upload_blob() -> None:
-        async with ContainerClient.from_connection_string(
+        async with ContainerClient.from_connection_string(  # type: ignore[attr-defined]
             _connection_string(subscription, resource_group, storage_name),
             container_name,
         ) as container_client:

--- a/webviz_config/_docs/_build_docs.py
+++ b/webviz_config/_docs/_build_docs.py
@@ -21,10 +21,10 @@ from typing import Any, Dict, Optional, Tuple, List
 
 try:
     # Python 3.8+
-    from typing import TypedDict  # type: ignore
+    from typing import TypedDict
 except (ImportError, ModuleNotFoundError):
     # Python < 3.8
-    from typing_extensions import TypedDict  # type: ignore
+    from typing_extensions import TypedDict
 
 import jinja2
 

--- a/webviz_config/_localhost_token.py
+++ b/webviz_config/_localhost_token.py
@@ -92,7 +92,7 @@ class LocalhostToken:
         def _set_cookie_token_in_response(
             response: flask.wrappers.Response,
         ) -> flask.wrappers.Response:
-            if flask.g.get("set_cookie_token", False):  # type: ignore[attr-defined]
+            if flask.g.get("set_cookie_token", False):
                 response.set_cookie(
                     key=f"cookie_token_{self._port}", value=self._cookie_token
                 )

--- a/webviz_config/generic_plugins/_markdown.py
+++ b/webviz_config/generic_plugins/_markdown.py
@@ -21,7 +21,7 @@ class _WebvizMarkdownExtension(Extension):
         super().__init__()
 
     def extendMarkdown(self, md: markdown.Markdown) -> None:
-        md.inlinePatterns.register(  # type: ignore[attr-defined]
+        md.inlinePatterns.register(
             item=_MarkdownImageProcessor(IMAGE_LINK_RE, md, self.base_path),
             name="image_link",
             priority=50,

--- a/webviz_config/plugins/__init__.py
+++ b/webviz_config/plugins/__init__.py
@@ -6,7 +6,7 @@ from typing import Optional
 
 try:
     # Python 3.8+
-    from importlib.metadata import distributions  # type: ignore
+    from importlib.metadata import distributions
 except ModuleNotFoundError:
     # Python < 3.8
     from importlib_metadata import distributions  # type: ignore

--- a/webviz_config/plugins/_utils.py
+++ b/webviz_config/plugins/_utils.py
@@ -4,11 +4,11 @@ from typing import Any, Dict, Iterable, Optional, Tuple
 
 try:
     # Python 3.8+
-    from typing import TypedDict  # type: ignore
-    from importlib.metadata import requires, version, PackageNotFoundError  # type: ignore
+    from typing import TypedDict
+    from importlib.metadata import requires, version, PackageNotFoundError
 except ImportError:
     # Python < 3.8
-    from typing_extensions import TypedDict  # type: ignore
+    from typing_extensions import TypedDict
     from importlib_metadata import requires, version, PackageNotFoundError  # type: ignore
 
 

--- a/webviz_config/themes/__init__.py
+++ b/webviz_config/themes/__init__.py
@@ -1,6 +1,6 @@
 try:
     # Python 3.8+
-    from importlib.metadata import entry_points  # type: ignore
+    from importlib.metadata import entry_points
 except ModuleNotFoundError:
     # Python < 3.8
     from importlib_metadata import entry_points  # type: ignore

--- a/webviz_config/utils/_localhost_open_browser.py
+++ b/webviz_config/utils/_localhost_open_browser.py
@@ -75,11 +75,11 @@ class LocalhostOpenBrowser:
         try:
             with urllib.request.urlopen(self._url()):  # nosec
                 app_ready = True
-        except urllib.error.HTTPError:  # type: ignore[attr-defined]
+        except urllib.error.HTTPError:
             # The flask instance responded with some HTTP error (likely 401),
             # but is otherwise ready to accept connections
             app_ready = True
-        except urllib.error.URLError:  # type: ignore[attr-defined]
+        except urllib.error.URLError:
             # The flask instance has not started
             app_ready = False
         except ConnectionResetError:

--- a/webviz_config/utils/_silence_flask_startup.py
+++ b/webviz_config/utils/_silence_flask_startup.py
@@ -26,4 +26,4 @@ def silence_flask_startup() -> None:
     def silent_function(*_args: Any, **_kwargs: Any) -> None:
         pass
 
-    flask.cli.show_server_banner = silent_function  # type: ignore[attr-defined]
+    flask.cli.show_server_banner = silent_function

